README.md

```diff
@@ -12,16 +12,16 @@ To install the library add:
       maven { url "https://jitpack.io" }
  }
  dependencies {
-      implementation 'com.github.AMPnet:jwt:0.1.5'
+      implementation 'com.github.AMPnet:jwt:1.0.0'
   }
  ```  
 
 ## Example
 Decode JWT payload: 
 ```json
 {
-  "sub": "e@mail.com",
-  "user": "{\"uuid\":\"370a248e-cbd0-4af8-b1d5-0851a3d654db\",\"email\":\"e@mail.com\",\"name\":\"Name\",\"authorities\":[\"Auth\"],\"enabled\":true,\"verified\":true,\"coop\":\"ampnet-demo\"}",
+  "sub": "AMPnet",
+  "address": "{\"userAddress\":\"0x745367860c5015B1E0AC04E00f1DbAd83B7dC272\"}",
```

Comment:
Why use object when there is only one value? Just use key address with string value.

Reply:
In case we add some additional information in the future and for encapsulation purposes. 
If you find it redundant I can remove the Address object.